sass/components/_categorical-display.scss

```diff
@@ -61,6 +61,56 @@
         border-bottom: none;
         padding: 0;
 
+        &__group {
+            &-icon {
+                margin-right: 4.5px !important;
```

Comment:
Why are we using `!important` in multiple places in this file?
Can you define another classname?

Reply:
It is already a new classname, it gets overwritten by generic `.ic-icon` styles because icon class is nested and has higher priority. I think it doesn't make sense to define another 2 new nested classnames inside this one just to override generic icon class, so I'm using important here instead.

This applies to other classes in this css, the generic styles are always nested and override the custom classes, so we need to refactor everything if we want to remove important from the file